{"name": "Запрос с несуществующим номером заказа возвращает ошибку", "status": "passed", "steps": [{"name": "Получение информации о заказе по трек-номеру", "status": "passed", "parameters": [{"name": "track_number", "value": "99999999"}], "start": 1735024816050, "stop": 1735024817135}], "start": 1735024816050, "stop": 1735024817136, "uuid": "27cc9035-df7b-4891-a24b-183a6658b729", "historyId": "39d6d4ca54158ef7e4a76431cc3f205c", "testCaseId": "39d6d4ca54158ef7e4a76431cc3f205c", "fullName": "tests.orders.test_get_orders.TestOrders#test_get_order_by_track_nonexistent_number", "labels": [{"name": "feature", "value": "Заказы"}, {"name": "parentSuite", "value": "tests.orders"}, {"name": "suite", "value": "test_get_orders"}, {"name": "subSuite", "value": "TestOrders"}, {"name": "host", "value": "MBP-Dmitrij.lan"}, {"name": "thread", "value": "2299-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.orders.test_get_orders"}]}